instrumentation/jmx-metrics/library/src/main/java/io/opentelemetry/instrumentation/jmx/engine/MetricBanner.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.jmx.engine;
+
+/**
+ * A class providing the user visible characteristics (name, type, description and units) of a
+ * metric to be reported with OpenTelemetry.
+ *
+ * <p>Objects of this class are immutable.
+ */
+public class MetricBanner {
```

Comment:
I'm not following the association with the word Banner?

Reply:
The class encapsulates metric properties which help identify/explain it to end users via a hypothetical UI, before looking at the metric values or other technical details. They represent the facet of the metric which will most likely be looked at first when browsing a list of metrics. Other possible names considered were `MetricTitle` or `MetricManifest`. What name would you propose?

Reply:
maybe `MetricInfo`?